### PR TITLE
Fix error, trying to get string on null object.

### PR DESCRIPTION
**Pull request by FeralBytes**

### CODE REVIEW

**Comment by don-tnowe** [comment]
When does it even happen, though?

File: addons/dictionary_inspector/inspector_plugin.gd [hunk 1]
```diff
@@ -24,8 +24,10 @@ func _can_handle(object):
 
 
 func _parse_property(object, type, path, hint, hint_text, usage, wide) -> bool:
-	if !typeof(object[path]) in supported_types:
-		return false
+	if object != null:
```
Comment (don-tnowe) [inline addons/dictionary_inspector/inspector_plugin.gd:27]: Indented every single line right there 🤔

How about `if object == null: return false` and keep everything else

Comment (FeralBytes) [reply addons/dictionary_inspector/inspector_plugin.gd:27]: Ok that sounds good, do you want me to make and update or just close this PR.

Comment (FeralBytes) [reply addons/dictionary_inspector/inspector_plugin.gd:27]: As far as when it happens, is on ,my project where I have multiple objects.

Comment (don-tnowe) [reply addons/dictionary_inspector/inspector_plugin.gd:27]: It's not too bad.

**Pull request merged**